web/testing/options-with-save/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    
+    <!-- Set the viewport width to match phone and tablet device widths -->                         
+    <meta name="viewport" content="width=device-width,user-scalable=no" /> 
+    
+    <!-- Enable IE9 Standards mode -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" /> 
+      
+    <title>KeymanWeb Testing Page - Basic Mnemonic Support</title>
+
+    <!-- Your page CSS --> 
+    <style type='text/css'>   
+      body {font-family: Tahoma,helvetica;}
+      h3 {font-size: 1em;font-weight:normal;color: darkred; margin-bottom: 4px}
+      .test {font-size: 1.5em; width:80%; min-height:30px; border: 1px solid gray;}
+      #KeymanWebControl {width:50%;min-width:600px;}       
+    </style> 
+ 
+    <!-- Insert uncompiled KeymanWeb source scripts -->              
+    <script src="../../release/unminified/web/keymanweb.js" type="application/javascript"></script>
+
+    <!-- 
+      For desktop browsers, a script for the user interface must be inserted here.
+       
+      Standard UIs are toggle, button, float and toolbar.  
+      The toolbar UI is best for any page designed to support keyboards for 
+      a large number of languages.
+    -->
+    <script src="../../release/unminified/web/kmwuitoggle.js"></script>
+ 
+    <!-- Initialization: set paths to keyboards, resources and fonts as required -->
+    <script>
+      var kmw=window.keyman;
+      kmw.init({
+        });
+    </script> 
+    
+    <!-- Add keyboard management script for local selection of keyboards to use -->
+    <!--<script src="../commonHeader.js"></script>-->
+  
+    <script>
+      kmw.addKeyboards({
+        id:'options_with_save',
+        name:'Options Test',
+        languages: {
+          id:'en',
+          name:'English'
+        },
+        filename:'./options_with_save.js'
+      });
+    </script>
+  </head>
+
+  <body>
+    <h2>KeymanWeb:  Test basic variable-store functionality</h2>
+    <p><ul>
+      <li><code>a</code>: Outputs "foo." if store is 'true', "no foo." if store is 'false'.</li>
+      <li><code>0</code>: Sets store to 'false'.</li>
+      <li><code>1</code>: Sets store to 'true'.</li>
+      <li><code>2</code>: "Saves" store; value should persist after a page reload.</li>
```

Comment:
On my setup, the test page behaves as if the store were `false` on page reload. (I'm having to also hit the "Use Web keyboard" button on each load. Otherwise a -> a).

Key sequence
a01a2
page reload
a --> no foo.

Reply:
ah, will rerun with the newest commits...

Reply:
I did note having to toggle the keyboard on after I cleared the page's cache from the past day (since before I started work on this issue).  Even with the purged web-cookie, it's still working correctly on my end.

Just in case, maybe try purging the cache/rebuilding?

Reply:
My Chrome must have latched on to something.
Rerunning in FF worked fine